### ISSUE TITLE
Should the library generator add a path to the workspace tsconfig.base.json?

### ISSUE THREAD

**Issue by samatechtw** [samatechtw/nx-vue3-vite#26]
The title describes the issue accurately. This came up when using the following command to rename a library generated by `nx-vue3-vite`:
```
> npx nx g @nrwl/workspace:mv --project=widgets --destination web/ui-widgets

unable to find "@monorepo/web/widgets" in tsconfig.base.json compilerOptions.paths
```

We don't use these aliases in generated apps due to how Vite is set up, but Nx expects them for renaming, and possibly for other features, so it's probably best to include.

**Comment by demisx** [comment]
I also think the generator should add that path alias to `tsconfig.base.json`, since Nx does it for other apps too.

**Comment by sampullman** [comment]
Fixed in #70
